pkg/tracing/traceparent_bindings.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 The Knative Authors
+
```

Comment:
<!--copyright-bot-->
Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

Reply:
alright the off-by one errors here should be resolved.  I've been working to switch to Github's new commenting scheme, which is required to take advantage of multi-line comments